### PR TITLE
chore: replace yarn with npm

### PR DESCRIPTION
**Pull request by Itrulia**

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -1,56 +0,0 @@
-# Editor
```
Comment (Itrulia) [inline .env.local:1]: Shouldn't have been commited

File: docs/README.md [hunk 2]
```diff
@@ -188,4 +188,7 @@ Within this event another invoice event will be triggered which is handled in `e
 To perform migrations on the database two steps are necessary.
 
 1. add an object in migrations.ts.
-2. restart the API (yarn watch)
+   <<<<<<< HEAD
```
Comment (Itrulia) [inline docs/README.md:191]: Merge conflict

File: docs/README.md [hunk 3]
```diff
@@ -188,4 +188,7 @@ Within this event another invoice event will be triggered which is handled in `e
...
+2. # restart the API (npm run watch)
+3. restart the API (yarn watch)
+   > > > > > > > master
```
Comment (tomaszdurka) [inline docs/README.md:194]: should this be here?

File: lerna.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "8.1.1",
+  "version": "10.0.0-alpha.1",
```
Comment (tomaszdurka) [inline lerna.json:2]: oh what's up with this?

Comment (Itrulia) [reply lerna.json:2]: That's from my testing :D

File: libs/api/__tests__/specs/peer.test.ts [hunk 5]
```diff
@@ -150,7 +150,6 @@ describe('Peers', () => {
     test('can update peer profile', async () => {
       const input: PeerProfileInput = {
         name: 'test peer profile',
-        logoID: 'logoID123',
```
Comment (Itrulia) [inline libs/api/__tests__/specs/peer.test.ts:153]: This was causing an error and the snapshot included it. We don't have any image uploads in tests so far so I deleted it for now.

File: .npmrc [hunk 6]
```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```
Comment (Itrulia) [inline .npmrc:1]: Without it we get an error because of @karma.run requiring React 16 in peerDeps

**Review by tomaszdurka — CHANGES_REQUESTED** [review]
I just found this one yarn in the project still.

```
https://github.com/wepublish/wepublish/blob/f/WPC-963-remove-yarn/config/create-gke-manifest.js
```

Apart from that looks great !

**Comment by Itrulia** [comment]
Good catch @tomaszdurka, updated

**Review by tomaszdurka — DISMISSED** [review]

File: helm/charts/wepublish/templates/deployment-api.yaml [hunk 7]
```diff
@@ -32,7 +32,7 @@ spec:
           image: {{ .Values.deployment.image }}
           imagePullPolicy: {{ .Values.deployment.imagePullPolicy }}
           command: ['/bin/sh']
-          args: ['-c', 'yarn prisma migrate deploy && npx ts-node libs/api/prisma/run-seed.ts && node dist/libs/api/src/runServer.js']
+          args: ['-c', 'npx prisma migrate deploy && npx ts-node libs/api/prisma/run-seed.ts && node dist/libs/api/src/runServer.js']
```
[the remark this answers is not in the document]
Comment (Itrulia) [reply helm/charts/wepublish/templates/deployment-api.yaml:35]: No, npx is for binaries such as prisma. npm run is for scripts inside the package.json

**Pull request merged**